### PR TITLE
EVA-966 - Don't require GPG signing when pushing dependency to clojars

### PR DESCRIPTION
**Pull request by tylerwilding-wk**
## Description of Changes

 - Previous Artifacts have been deployed to Clojars without GPG signing, this overrides the default `lein deploy clojars` profile to not use GPG signing

## Proposed Testing Steps (If Applicable)

  -

## Relevant Issues (If Applicable)

  -

## Maintainer Notifications

  -

### CODE REVIEW

**Comment by aviary2-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://dev.workiva.net/docs/teams/information-security/aviary#configuring-aviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: README.md [hunk 1]
```diff
@@ -2,11 +2,6 @@
 
 > *utiliva, compound Latin substantive adjective* - Things for doing useful stuff
 
-```
-[com.workiva/utiliva "0.2.0"]
```
Comment (danielharasymiw-wf) [inline README.md]: Should this be removed? I'm a fan of copy pasting these things to add them to my project.

Comment (tylerwilding-wk) [reply README.md]: Well in my opinion yes, we have an automatically updating shield now 
image

**Comment by danielharasymiw-wf** [comment]
+1

**Comment by tylerwilding-wk** [comment]
@Workiva/release-management-p

**Pull request merged**